src/traversals/maxadjvisit.jl

```diff
@@ -83,16 +83,16 @@ displayed.
 function maximum_adjacency_visit(
     g::AbstractGraph,
     distmx::AbstractMatrix{T},
-    log::Bool,
-    io::IO
+    log::Bool=false,
+    io::IO=STDOUT
 ) where T<:Real
 
     U = eltype(g)
     pq = DataStructures.PriorityQueue{U, T}(Base.Order.Reverse)
     vertices_order = Vector{U}()
     has_key = ones(Bool, nv(g))
     sizehint!(vertices_order, nv(g))
-    @assert nv(g) >= 2
+    nv(g) >= 2 || error("nv(g) must be greater than 1")
```

Comment:
This is good, because we are replacing an assertion error with a regular error, but I think in general returning an empty output for graphs with `nv==1` is better than raising an exception. You might reasonably have a list of graphs of various sizes, and some could be `nv==1`. Everything should work without exceptions producing a sensible meaningless value such as empty array, or array with only one element rather than introducing a try catch block around every method call.

But for consistency with existing API, I think this is fine for now.

Reply:
You're right. I'm going to change this now. I don't like errors.